frontend/src/features/admin-form/AdminFormResultsFeedbackPage.stories.tsx

```diff
@@ -69,10 +72,23 @@ const Template: Story = () => {
 }
 export const Default = Template.bind({})
 
-export const EmptyFeedback = Template.bind({})
-EmptyFeedback.parameters = {
+export const EmptyReviewAndIssue = Template.bind({})
+EmptyReviewAndIssue.parameters = {
+  msw: [
+    getEmptyAdminFormIssue(),
+    getEmptyAdminFormFeedback(),
+    ...DEFAULT_MSW_ROUTES,
+  ],
+}
+
+export const EmptyReview = Template.bind({})
+EmptyReview.parameters = {
   msw: [getEmptyAdminFormFeedback(), ...DEFAULT_MSW_ROUTES],
 }
```

Comment:
Just for clarification, do you mean the EmptyIssue? There is not route for Review tab, so the default page will always be Issue tab.

Reply:
Ah okay. In this case, maybe we can change the frontend test to test the issues tab component itself and the review tab component itself. Or set up anchoring for issue / review tab so they have routes. Can be done in a separate PR to prevent this PR from getting too big 👍

Reply:
Actually how does the frontend test works? Do you mean if the tab is not showing then the chromatic diff will not do the diff for us? Yeah, i wanted to create a new route initially but the changes will be much bigger, wanted to minimise the refactoring to aim for quicker launch.

Reply:
Yea if you don't see it rendered on chromatic, chromatic won't see it either, so it won't be able to compare the frontend stuff for us.
So to avoid adding routes just to test the UI, we can render the component itself and test the UI. For e.g. this is what we do for the [builder logic tab](https://www.chromatic.com/component?appId=60911f69d4403b0039b266a2&csfId=pages-adminformpage-create-logictab&buildNumber=4932&k=64abdba17e2b4d1dae169572-1200-interactive-true&h=4&b=-1).
Yea but as I mentioned, can leave this as separate issue for quicker launch! Rmb to assign Stacey as reviewer for the new tests that are added for the review tab frontend tests PR 👍